### ISSUE TITLE
Inference using faster_rcnn_inception_resnet does not draw the correct position of the bounding boxes.

### ISSUE THREAD

**Issue by lozuwa** [tensorflow/models#4310]
### System information
- **What is the top-level directory of the model you are using**: 
models/research/
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Ubuntu 16.04
- **TensorFlow installed from (source or binary)**:
Binary
- **TensorFlow version (use command below)**:
1.5
- **Bazel version (if compiling from source)**:
No
- **CUDA/cuDNN version**:
Cuda 9.0. Cuddn 7.0.
- **GPU model and memory**:
GTX 1080 ti 11 Gb
- **Exact command to reproduce**:
It is not an issue with a command.

### Describe the problem
Hi, I trained faster_rcnn_inception_resnet from scratch with my own dataset. Then, I exported an inference graph and run the tutorial on object detection to try the inference. Everything works fine except for the position of the bounding box when using:
```python
vis_util.visualize_boxes_and_labels_on_image_array(image_np,
                                                            output_dict['detection_boxes'],
                                                            output_dict['detection_classes'],
                                                            output_dict['detection_scores'],
                                                            category_index,
                                                            min_score_thresh=.8,
                                                            instance_masks=output_dict.get('detection_masks'),
                                                            use_normalized_coordinates=True,
                                                            line_thickness=12)
```
I get the image shown below. This happens for all the classes and all the images. I also have the following coordinates from the output dictionary which of course are not correct so it is not a problem with visualize_boxes_and_labels_on_image_array.

```python
print(output_dict['detection_scores'][:2])
print(output_dict['detection_boxes'][:2])
```

[0.9800197 0.5276416]
[[0.03427145 0.04767951 0.1516928  0.19400768]
 [0.01404628 0.02613962 0.18203202 0.2045249 ]]

![img26](https://user-images.githubusercontent.com/11334414/40261083-b43501a4-5acc-11e8-893a-99174015d5f7.jpg)

By the way, all of the images in my training set are width=1000 and height=600. The same for the validation set.

**Comment by nguyeho7** [comment]
A quick shot from the hip, but try to multiply the x_min/x_max coordinates with the width and y_min/y_max coordinates with the height

**Comment by lozuwa** [comment]
Yeah, I have already tried that. I also tried to find a scaling factor by approximating a function. But there doesn't seem to be correlation. I would say the RPN is not well optimized, but it had already run more than 500K iterations. 
My other guess would be the image annotations, maybe I have done something wrong in the creation of the TF record.
